src/common/models/project.go

```diff
@@ -15,9 +15,13 @@
 package models
 
 import (
+	"fmt"
```

Comment:
can we move this into pkg/project/model?

Reply:
The project-related models still used by functions in `common/dao`, these models will be moved to `pkg/project/models` when the functions about the project in the `common/dao` were removed.